proto/pdpb.proto

```diff
@@ -240,6 +240,10 @@ message RegionHeartbeatRequest {
     uint64 keys_read = 9;
     // Approximate region size.
     uint64 approximate_size = 10;
+    // Pengding Learner peers are the non-voter peers that are catching up with leader's log
+    repeated metapb.Peer pending_learner_peers = 11;
+    // Pengding Learner peers are the non-voter peers that are waiting to be promoted to voter
+    repeated metapb.Peer complete_learner_peers = 12;
```

Comment:
Does we really need this? We can got these learners by `setdiff(region.learners, pending_learners)`.

Reply:
And I think you can also add `repeated Peer learners` in `metapb::Region` in this PR. @BusyJay what do you think?

Reply:
get it.